featurebyte/session/base.py

```diff
@@ -536,6 +536,9 @@ async def execute_query(
         try:
             bytestream = self.get_async_query_stream(query=query, timeout=timeout)
         except Exception as exc:
+            logger.error(
+                "Error executing query", extra={"query": query, "source_type": self.source_type}
+            )
```

Comment:
There are some queries that are expected to fail such as those that checks whether a table exists by trying to select from the table. Would be good if there is a way to exclude those from the logs.

Reply:
Indeed. Would there be any way to identify these statements?

Reply:
I think one way is to allow execute_query's caller to specify whether failure is expected through a parameter, and update places like this: https://github.com/featurebyte/featurebyte/blob/6de5d317741e1ce749faf5da2a46545e81e5768a/featurebyte/session/base.py#L802